bin/add-deadlines-to-legacy-tasks

```diff
@@ -36,20 +38,27 @@ const patchTaskDeadline = async (taskId, transaction) => {
         return {}; // submission was not complete and correct, reset deadline
       }
     }
-    if (activity.eventName === 'update') { // possible deadline extend event
+    if (activity.eventName === 'update' && !isEmpty(deadline)) { // possible deadline extend event
```

Comment:
This is to prevent the partial deadlines getting added (e.g. `{"isExtended":false,"isExtendable":true}`).